content/blogposts/2022/shift-left-wtf-does-it-mean.md

```diff
@@ -0,0 +1,235 @@
+1. User value—are you building something the user actually wants?
+2. Context—does it fit well into the broader system and take advantage of what that system provides?
+3. Testing—does it actually do what you say it does correctly and reliably?
+4. Security—does it preserve the privacy and integrity of the overall system?
+5. Review—does it pass muster with the folks who must approve for code quality and the overall maintenance of the system?
+
+The importance of these validation areas can be highlighted by their failure modes, which are all too familiar:
+1. The Over-Engineered Boondoggle that doesn't actually solve a real problem
+2. The Reinvented Wheel, which takes longer to build and doesn't roll that well
+3. The Buggy Launch that ends up frustrating rather than delighting users
+4. The Security Breech and the subsequent disruptive emergency response
+5. The Back to the Drawing Board that must be rewritten after a major flaw or inconsistency is caught only in final review
+
+To prevent these failure scenarios, you could just introduce new gatekeepers. But the problem is you already have too many gatekeepers, and gatekeepers can only save you from shipping a shoddy system. They can't get you to what you truly desire, which is to ship a good system on schedule.
+
+The real solution is to "shift left" as many of these validation points as possible, which means empowering the developer driving the project to validate these points much earlier than the formal review step.
+
+## Shifting left user value validation
+
+It seems like an obvious statement, but a surprising number of software projects make it very far along before validating that the deliverable actually delivers value to the end user. Shifting left the user validation means getting to an MVP that can be shared with users as quickly as possible. This lets you solicit high-quality feedback, which is critical because chances are that your initial idea does not fully address the user's need.
+
+Time is of the essence. Discovering existing functions and functionality early on—and avoiding the temptation to reinvent the wheel—can save loads of time.
+
+<center>
+  <video title="Code reuse" alt="Discovering and reusing existing code can help you spin up a MVP ASAP, so you can quickly validate the user need with a rough sketch of the product." loop autoPlay muted playsInline>
+    <source src="https://storage.googleapis.com/sourcegraph-assets/blog/shift-left/01-code-reuse.mp4" type="video/mp4" data-cookieconsent="ignore" />
+    <source src="https://storage.googleapis.com/sourcegraph-assets/blog/shift-left/01-code-reuse.webm" type="video/webm" data-cookieconsent="ignore" />
+  </video>
+  <figcaption>
+    Discovering and reusing existing code can help you spin up a MVP ASAP, so you can quickly validate the user need with a rough sketch of the product.
+  </figcaption>
+</center>
```

Comment:
This will ease and conform our video implementation by miles. What do you think of adding a prop for alt text as well? This way, we can bake in that extra step for accessibility.

Also, let's go with your observation on the `article`! This way we can prune out these extra elements.

Reply:
1. This will ease and conform our video implementation by miles. I created an issue to track this,  https://github.com/sourcegraph/about/issues/5496, and updated the Shift Left post and other About pages with the `Video` and `YouTube` components: 495762c. 

Note: Adding `YouTube` to the Community page happens to resolve [#5448](https://github.com/sourcegraph/about/issues/5448). There is more cleanup to do, but this will cover what's connected with this blogpost. I will continue tracking cleanup in the issue!

2. Let's go with your observation on the `article`, too. This way we can prune out these extra `center` elements. Please see: be30900.